middleware/rewrite/README.md

```diff
@@ -54,6 +55,21 @@ rewrites the first local option with code 0xffee, setting the data to "abcd". Eq
 rewrite edns0 local set 0xffee abcd
 ~~~
 
+* A variable data must start with `$`. Following are the supported variables:
+    * $qname
+    * $qtype
+    * $client_ip
+    * $client_port
+    * $protocol
+    * $server_ip
+    * $server_port
```

Comment:
Variables in templates in other middleware (*auto* and *reverse*) are using the `{}` syntax, so `$qname` would become `{qname}`. etc.